tools/docker_dev/.gitpod.Dockerfile

```diff
@@ -1,8 +1,16 @@
+# Doing a local shallow clone - keeps the container secure
+# and much slimmer than usins COPY directly
```

Comment:
minor: typo in `usins`

Reply:
Out of interest: why is it more secure?

Reply:
ah I deleted a bit, should be:

`# and much slimmer than using COPY directly or remote clone`

that makes more sense: more secure than the remote clone as you do not pass tokens or have to persist keys with buildkit

Reply:
makes sense now, thanks